.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "src/components/lib/ps"]
 	path = src/components/lib/ps
 	url = https://github.com/gwsystems/ps.git
+[submodule "src/extern/cFE"]
+	path = src/extern/cFE
+	url = https://github.com/Others/cFE.git
```

Comment:
Should this be sourced from gwsystems instead?

Reply:
No. This is not the cFE repository on gwsystems. (Which really should be changed back to `cFE2cos`...) This cFE repo just has cFE code in it.